### DISCUSSION TITLE
Unable to connect Snowflake to Superset

### DISCUSSION THREAD

**Discussion by Sypher1k** [apache/superset#16010] [Q&A / Help]
I have installed Superset(1.2) on Ubuntu 20.x,
I'm trying to connect to Snowflake but the below error comes up,

![Screenshot 2021-08-02 084116](https://user-images.githubusercontent.com/20921224/127799575-5e18d727-21fa-4555-a00c-f83afe80f272.jpg)

I have installed the required drivers already,

![Screenshot 2021-08-02 083338](https://user-images.githubusercontent.com/20921224/127799653-66ea9cf1-bcd8-4631-920b-4b27d0448508.jpg)

Can someone help me with this please?

**Comment by nytai** [comment]
Are you sure the sqlalchemy package has been installed in the same context as superset?
